draft-dekater-scion-controlplane.md

```diff
@@ -1301,7 +1302,7 @@ A PCB originated by a given control service is validated by all the control serv
 * A fast clock at origination or a slow clock at reception will yield a lengthened expiration time for hops, and possibly an origination time in the future.
 * A slow clock at origination or a fast clock at reception will yield a shortened expiration time for hops, and possibly an expiration time in the past.
 
-This bias comes in addition to a structural delay: PCBs are propagated at a configurable interval (typically, around one minute). As a result of this and the way they are iteratively constructed, PCBs with N hops may be validated up to N intervals (so maximally N minutes) after origination. This creates a constraint on the expiration of hops. Hops of the minimal expiration time (337.5 seconds - see [](#hopfield)) would render useless any PCB describing a path longer than 5 hops. For this reason, it is unadvisable to create hops with a short expiration time, that should be around 6 hours.
+This bias comes in addition to a structural delay: PCBs are propagated at a configurable interval (typically, around one minute), except at AS bootstrapping. As a result of this and the way they are iteratively constructed, PCBs with N hops may be validated at worst case up to N intervals (so maximally N minutes) after origination. This creates a constraint on the expiration of hops. Hops of the minimal expiration time (337.5 seconds - see [](#hopfield)) would render useless any PCB describing a path longer than 5 hops. For this reason, it is unadvisable to create hops with a short expiration time, that should be around 6 hours.
```

Comment:
Same consideration about boostrapping. What is bootstrapping.

Reply:
Ok, I will remove all mentions of bootstrapping. However, I'd like that we discuss (maybe also with oncilla) if this fast retry mechanism should really be optional, or perhaps mandatory. 
If it stays optional (using MAY in comment above), then we are ok to say here `at worst case`. If it becomes mandatory, then the structural delay here might become a lot better (however, I feel we still don't understand the full implications).

Reply:
(fee free to resolve if you like my text, or to propose an alternative)